android/app/src/main/kotlin/com/nt4f04und/sweyer/MusicPlayerAppWidget.kt

```diff
@@ -260,7 +260,7 @@ object AudioServiceBackgroundIntent {
      * Build a pending intent for the given [context] emulating a key-press of the given [keyEvent] to the AudioService.
      */
     private fun getIntent(context: Context, keyEvent: KeyEvent): PendingIntent {
-        val intent = Intent(context, com.ryanheise.audioservice.AudioService::class.java)
+        val intent = Intent(context, com.ryanheise.audioservice.MediaButtonReceiver::class.java)
```

Comment:
Are we sure this will work if the services is not started?

Reply:
I now see your other comment https://github.com/nt4f04uNd/sweyer/pull/144#discussion_r1809737134